test/test.js

```diff
@@ -683,6 +683,20 @@ suite('SystemJS Standard Tests', function() {
     });
   });
 
+  test('Source maps', function () {
+    return System.import('tests/x-error.js')
+    .catch(function (error) {
+      ok(error.stack.includes('x-error.js'));
+    });
+  });
+
+  test('Locate plugin source maps', function () {
+    return System.import('tests/x-error!tests/locate-plugin.js')
+    .catch(function (error) {
+      ok(error.stack.includes('x-error.js'));
+    });
+  });
```

Comment:
This makes sure the locate-plugin's address is the address in the stacktrace. Before my change, this test errors.